src/zenml/zen_stores/migrations/versions/3b68abe58f44_add_model_entity.py

```diff
@@ -11,7 +11,7 @@
 
 # revision identifiers, used by Alembic.
 revision = "3b68abe58f44"
-down_revision = "0.44.2"
+down_revision = "0.44.3"
```

Comment:
why did this happen? because we since released a new version?

Reply:
Yes, cause there was a release in between and this branch had 2 alembic branches, which is not supported in our case.